bee-network/bee-autopeering/src/server.rs

```diff
@@ -259,14 +265,14 @@ impl Runnable for OutgoingPacketHandler {
                         let bytes = packet.to_protobuf();
 
                         if bytes.len() > MAX_PACKET_SIZE {
-                            log::warn!("Trying to send too many bytes to {}. Ignoring...", peer_addr);
+                            log::warn!("Trying to send too many bytes to {}. Ignoring packet.", peer_addr);
                             continue 'recv;
                         }
 
-                        // TODO: Make sure this won't occur by introducing IPv4 and IPv6 outgoing packet handler.
                         let n = outgoing_socket.send_to(&bytes, peer_addr).await.expect("socket send error");
 
                         log::trace!("Sent {} bytes to {}.", n, peer_addr);
+
```

Comment:
Is this newline intentional here?

Reply:
could be, but I am not too opinionated on it. `rustfmt` was fine with it, so I guess there are people who actually prefer the blank line in such a case. Tbh.... the `} else {` is indeed (but only very little) disturbing the readability of this line, but .... I really don't care that much.

Removed.